models/staging/jaffle_shop/src_jaffle_shop.yml

```diff
@@ -0,0 +1,31 @@
+version: 2
+
+sources:
+    - name: stripe
+      database: raw
+      tables:
+        - name: payment

```

Comment:
To make this easier to read, I recommend adding a space between the stripe sources and jaffle shop.